README.md

```diff
@@ -43,6 +43,11 @@ any contributions.
 
 For more, check out the [Contributing Guide](CONTRIBUTING.md).
 
+## Maintained
+
+Maskito is a part of [Taiga UI](https://github.com/Tinkoff/taiga-ui) libraries family which is backed and used by a large enterprise.
+This means you can rely on timely support and continuous development.
```

Comment:
See https://github.com/Tinkoff/maskito/actions/runs/4949047477/jobs/8850618639?pr=295

```
Run npm run typecheck

> maskito@0.[11](https://github.com/Tinkoff/maskito/actions/runs/4949047477/jobs/8850618639?pr=295#step:4:12).1 typecheck
> tsc --noEmit --skipLibCheck --incremental false --tsBuildInfoFile null --project tsconfig.spec.json


> maskito@0.11.1 format
> prettier '**/*.{svg,yml,js,jsx,ts,tsx,html,md,less,json}' --check

Checking formatting...
[warn] README.md
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
```

Reply:
Can we autorun prettier and lint fix on PRs from members of the organization, like we do in Taiga UI?

Reply:
I've created this task for this:
* https://github.com/Tinkoff/maskito/issues/297